openmc/tallies.py

```diff
@@ -1653,7 +1659,8 @@ def _align_tally_data(self, other):
             if self_repeat_factor == 1:
```

Comment:
@wbinventor do you know why there is the fork for the tile and repeat here? When aligning for nuclide and score the other tally is always tiled.